src/resources/js/buttonset.js

```diff
@@ -0,0 +1,89 @@
+var tribe_buttonset = tribe_buttonset || {};
+
+( function( $, obj ) {
+	'use strict';
+
+	obj.$body;
+
+	obj.selector = {
+		buttonset: '.tribe-buttonset',
+		button: '.tribe-button-field',
+		input: '.tribe-button-input',
+		active: '.tribe-active'
+	};
+
+	obj.ready = function( event ) {
+		obj.$body = $( 'body' );
+		obj.$body.on( 'click.tribe_buttonset', obj.selector.button, obj.click );
+		obj.$body.on( 'change.tribe_buttonset', obj.selector.input, obj.change ).find( obj.selector.input ).trigger( 'change' );
```

Comment:
Optional - could move the `.find().trigger()` portion to a new line - as together they form a distinct operation in terms of setting up the initial UI vs the preparatory step of setting up the event listener.

Plus it would shorten the line a little, it exceeds our soft limit of 120chars atm.